pkgs/applications/window-managers/wideriver/default.nix

```
@@ -0,0 +1,30 @@
+{stdenv, fetchFromGitHub, lib, gcc, wayland, wayland-scanner, pkg-config }:
+
+stdenv.mkDerivation rec {
+  pname = "wideriver";
+  version = "1.1.0";
+
+  src = fetchFromGitHub {
+    owner = "alex-courtis";
+    repo = pname;
+    rev = version;
+    sha256 = "sha256-Jn3DfflDlMSZMlSRBfzQeEUKgYLctd/PiQRWH8uwtlU=";
+  };
+
+  buildInputs = [ gcc wayland-scanner wayland pkg-config ];
+
+  installPhase = ''
+    export DESTDIR=$out
+    export PREFIX=""
+
+    make install
+  '';
+
+  meta = with lib; {
```

```suggestion
  meta = {
```
https://github.com/NixOS/nixpkgs/issues/292468